crispy_forms/base.py

```diff
@@ -9,14 +19,19 @@ class KeepContext:
     touch context object themselves, that could introduce side effects.
     """
 
-    def __init__(self, context, keys):
+    def __init__(self, context: Context, keys: Collection[int | str | Node]) -> None:
```

Comment:
`int | str | Node` can be replaced with `_ContextKeys`

https://github.com/typeddjango/django-stubs/blob/79276804b49f98b64f2c627f995832b469dc0338/django-stubs/template/context.pyi#L12